src/renderer/components/ChatView.js

```diff
@@ -174,13 +174,35 @@ class ChatView extends React.Component {
     this.setState({ composerSize: size })
   }
 
+  onDrop (e) {
+    const files = e.target.files || e.dataTransfer.files
+    const { chat } = this.props
+    e.preventDefault()
+    e.stopPropagation()
+    // TODO maybe add a clause here for windows because that uses backslash instead of slash
+    const forbiddenPathRegEx = /DeltaChat\/[\d\w]*\/db\.sqlite-blobs\//gi
+    for (let i = 0; i < files.length; i++) {
+      const { path } = files[i]
+      if (!forbiddenPathRegEx.test(path)) {
```

Comment:
The whole RegEx thing is a workaround because I found no better solution.

Reply:
Not sure if this is needed. I get the point with not sneaking out the deltachat passwords and stuff, but if we are this far, that someone can control this part of the code, the evil person could also sneak out your `/etc/passwd` or your `~/porn collection`. Not sure if this really helps.

Reply:
Not thats not the point, this part prevents draging out a file from dc and drop it into dc and so sending it again by accident

Reply:
Aaah i get it :)